draft-ietf-quic-transport.md

```diff
@@ -3100,9 +3100,11 @@ Further validation can also be provided:
   use for validation (for example, the IP ID or UDP checksum).
 
 The endpoint SHOULD ignore all ICMP messages that are not validated or do not
-carry sufficient quoted packet payload to perform validation.  Any reduction in
-the QUIC maximum packet size MAY be provisional until QUIC's loss detection
-algorithm determines that the quoted packet has actually been lost.
+carry sufficient quoted packet payload to perform validation.  ICMP messages
+that suggest that the PMTU has decreased below 1280 bytes SHOULD also be
```

Comment:
Why only a SHOULD? We say that QUIC depends on a PMTU of at least 1280 bytes, so I think a MUST would be justified here.

Reply:
Oops, forgot to checkin my second pass.  I had exactly the same thought.